templates/Email/MFA/Notification_backupcodeused.ss

```diff
@@ -0,0 +1,10 @@
+<p><%t SilverStripe\\MFA\\Email\\Notification.HELLO 'Hi' %> $Member.Name,</p>
+
+<p>
+	<%t SilverStripe\\MFA\\Email\\Notification.USEDBACKUPCODE 'A recovery code was used to gain access to your account at' %> $AbsoluteBaseURL.<br />
```

Comment:
* It's probably worth including the link as part of the translation, in case RTL languages need to put it first
* I think the <br> could just be two separate paragraph tags

Reply:
hahaha markdown converts HTML 😂

Reply:
these changes were made.